src/Microsoft.OData.Client/DataServiceClientFormat.cs

```diff
@@ -275,4 +283,89 @@ private static string ChooseMediaType(bool hasSelectQueryOption)
             return MimeApplicationJsonODataLight;
         }
     }
+
+    /// <summary>
+    ///This class is a utility to help with POCO loading it can be registered as an option to load the CSDL over network or from a file or string
+    ///
+    /// </summary>
+    public class DataServiceClientFormatUtil

```

Comment:
where do you use this class? I mean why do you add this class?

Reply:
Removed after refactoring